ONSdigital/dis-authentication-stub
# Feature/login post handler

### What
Following endpoints added

- GET /florence/login
- POST /florence/login

### How to review

- Clone the repo
- Checkout to branch
- Build docker stack (dataset-catalogue)
- check if you can run POST /florence/login in the local host and create the cookies (access_token, id_token, refresh_token)

### Who can review

@franmoore05
